### DISCUSSION TITLE
Add Community Centre as building type

### DISCUSSION THREAD

**Discussion by lebeno** [streetcomplete/StreetComplete#4840] [Q&A] [answered]
Streetcomplete asks to indicate building type but I find no suiting suggestions for Tag:amenity=community_centre

Maybe this option can be added?

**Comment by matkoniecz** [comment]
Note that StreetComplete asks about building architecture type, not POI containing it.

For example church turned into a museum is still `building=church`

I would not expect Community Centre to be a distinct building typology.

**Reply by mnalis** [reply]
Perhaps me might consider renaming the question from `What kind of building is this?` to `What is architectural style of this building?` or something similar?

**Reply by mnalis** [reply]
see also SC FAQ ["Building Type Quest - Why don't I see the right types?"](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Building_Type_Quest_-_Why_don%27t_I_see_the_right_types%3F)

**Comment by lebeno** [comment, marked as answer]
Thank you for the info. I guess `building=public` is the most suitable answer in my case.

**Reply by matkoniecz** [reply]
Changed to QA to mark it as visibly answered in discussion list, I hope that it is OK.
